src/shapes.cpp

```diff
@@ -39,76 +39,103 @@
 #include <octomap/octomap.h>
 #include <console_bridge/console.h>
 
-const std::string shapes::Sphere::STRING_NAME = "sphere";
-const std::string shapes::Box::STRING_NAME = "box";
-const std::string shapes::Cylinder::STRING_NAME = "cylinder";
-const std::string shapes::Cone::STRING_NAME = "cone";
-const std::string shapes::Mesh::STRING_NAME = "mesh";
-const std::string shapes::Plane::STRING_NAME = "plane";
-const std::string shapes::OcTree::STRING_NAME = "octree";
-
-shapes::Shape::Shape()
+namespace shapes {
+
+const std::string Sphere::STRING_NAME = "sphere";
```

Comment:
How about using a `static const std::map<ShapeType, std::string>` ?

C++11 allows for this:
```
std::map <int, std::string> x { 
  { 42, "foo" }, 
  { 3, "bar" } 
};
```